trello_api.py

```diff
@@ -0,0 +1,92 @@
+from flask import session
+from classes import todoStatus, todoItem
+import os, requests, json, logging
+
+API_PREFIX = 'https://api.trello.com/1/'
+
+API_KEY = os.getenv('API_KEY')
+API_TOKEN = os.getenv('API_TOKEN')
+# API_PARAMS was getting appended with additional parameters, so had to set variable in each API function
+API_PARAMS = {'key': API_KEY, 'token': API_TOKEN}
```

Comment:
When you write `postParams = API_PARAMS`, you have two variables that are actually just different names for the same dictionary. They are pointing to the exact same object. 

What you wanted was `params = API_PARAMS.copy()`, then you can delete all the duplicated `API_PARAMS = ...` lines